### DISCUSSION TITLE
Possible bug: volume and constrast buttons restart the Windows Explorer App

### DISCUSSION THREAD

**Discussion by Alecampoy** [valinet/ExplorerPatcher#810] [General]
Hi, I'm finding this issue since the last windows update using this wonderfull app. Anytime I use the keys to change volume or contrast, the taskbar restars (althougt it changes the contrast)
Best wishes

**Comment by highlanderc** [comment]
This happens because W11 is not popping up its own flyover and thus it crashes. Same thing with the wi-fi. If you want the wi-fi flyover to work you need to select w11.

**Reply by microhobby** [reply]
interesting was that this was working before the last update and with the W10 selected on Taskbar style.

**Reply by valinet** [reply]
What Windows build are you on? It works fine on 434 for me, that’s the latest public stable build.

**Reply by microhobby** [reply]
@valinet yeah I'm on 543, now I updated to the latest pre release and it's working as expected, thanks for the attention!

**Reply by Alecampoy** [reply]
I'm using bould 22533
Found  on some forum that it is caused by this
https://answers.microsoft.com/en-us/insider/forum/all/local-language-experience-pack-is-the-cause-of-2/81482b4d-7445-4539-a140-072a4a693ca6

I unistall and now it is working fine

**Reply by highlanderc** [reply]
I am on build 22543 with latest pre release and the issue is rather random. Sometimes it will bring back the OSD for windows 11 and it all works fine. I did a restart of explorer and went back to crashing when either volume up or down or screen brightness up or down.

**Reply by valinet** [reply]
It’s kind of expected for the new OSDs to be broken, besides them replacing the Windows 8-era ones, I am pretty sure no one at Microsoft tests them against ExplorerPatcher or similar tweaks or even cares about them working with the legacy UI. We will hopefully fix them once the implementation stabilizes a bit, i.e. a new major update ships.

**Reply by highlanderc** [reply]
donated.... good job with this. Quick Launch is hard to lose. Don't get why Microsoft is SO HELL BENT in changing the damn os, instead of improving it.

**Reply by novatiberium** [reply]
Welp, I just updated ExplorerPatcher today to another release and the flyouts now crash EP again.

**Reply by Poopooracoocoo** [reply]
@novatiberium you might want to leave the windows insider program

**Reply by valinet** [reply]
What’s the surprise? You updated EP on the same Insider Preview build with the new flyouts and it still does not work? How could it? Do the changelog or commits mention anything about that? If not, then no one worked on that, and it cannot fix itself automagically.
